cap.scad

```diff
@@ -3,12 +3,43 @@ use <threadlib/threadlib.scad>
 // height of the cap (without thread)
 cap_height = 3; // [0:0.1:20]
 // diameter of the cap
-cap_diameter = 52; // [46:0.5:80]
+cap_diameter = 93; // [46:0.5:90]
```

Comment:
If we want to support wider caps, we should also update the comment, which actually affects the customizer's behavior.

Maybe something like `// [46:0.5:100]`.